Detectors/Upgrades/ALICE3/PSR/README.md

```diff
@@ -0,0 +1,13 @@
+<!-- doxy
```

Comment:
Update for PSR please

Reply:
> Update for PSR please
@mconcas ,I have applied the patch, deleted the unnecessery file made all the required changes. However, I am not sure what to do about these lines. Is it okay if I just keep the description of the Pre-Shower detector here? 

Once done, I plan to make a commit, merge it with teh initial commit and push it again to my repo.

Reply:
Yo don't need to merge any commit, just put the fixing commit on top.